src/components/Municipality/MunicipalityIntro/index.js

```diff
@@ -0,0 +1,172 @@
+      {!!municipality ? `${municipality.name}` : 'deiner Gemeinde'}! Hilf uns,
+      das Grundeinkommen in{' '}
+      {!!municipality ? `${municipality.name}` : 'deiner Gemeinde'} zu testen,
+      vernetze dich mit Gleichgesinnten und bleib auf dem Laufenden!
+    </p>
+  );
+};
+
+const BodyQualifing = ({ municipality, type, handlePlaceSelect }) => {
+  return (
+    <>
+      <SearchPlaces
+        placeholder={!!municipality ? municipality.name : 'Gemeinde'}
+        label={false}
+        onPlaceSelect={handlePlaceSelect}
+        showButton={true}
+      />
+      <MunicipalityHeadline
+        className={s.headline}
+        municipality={municipality}
+        type={type}
+      />
+      {!!municipality && (
+        <CampainVisualisation
+          goal={5000}
+          currentCount={1000}
+          receivedCount={2000}
+          count={3000}
+          // showCTA={visualisations.length !== 1 && visualisation.ctaLink}
+          labels={{
+            NEEDED: () => <>Benötigte Anmeldungen</>,
+            GOAL_INBETWEEN_TOOLTIP: count => (
+              <>
+                Insgesamt benötigt:
+                <br />
+                {count} Anmeldungen
+              </>
+            ),
+            GOAL_INBETWEEN: count => <>Nächstes Ziel: {count} Anmeldungen</>,
+            CURRENT_COUNT: () => <>Anmeldungen</>,
+            CTA: () => <>Mitmachen</>,
+          }}
+          currency="Anmeldungen"
+          startDate={new Date()}
+        />
+      )}
+      <EngagementText municipality={municipality} />
+      <SignUp />
+    </>
+  );
+};
+
+const BodyCollecting = ({ municipality, type, handlePlaceSelect }) => {
+  return (
+    <>
+      <MunicipalityHeadline
+        className={s.headline}
+        municipality={municipality}
+        type={type}
+      />
+      <SearchPlaces
+        placeholder={!!municipality ? municipality.name : 'Gemeinde'}
+        label={false}
+        onPlaceSelect={handlePlaceSelect}
+        showButton={true}
+      />
+    </>
+  );
+};
+
+const BodyState = ({ municipality, type, handlePlaceSelect }) => {
+  return (
+    <>
+      <MunicipalityHeadline
+        className={s.headline}
+        municipality={municipality}
+        type={type}
+      />
+      <SearchPlaces
+        placeholder={!!municipality ? municipality.name : 'Gemeinde'}
+        label={false}
+        onPlaceSelect={handlePlaceSelect}
+        showButton={true}
+      />
+    </>
+  );
+};
+
+const MapHeader = ({ municipality }) => {
+  return (
+    <div className={s.headerContainer}>
+      <CampaignMap
+        className={s.mapContainer}
+        focusAGS={!!municipality ? municipality.ags : undefined}
+        initialAnimation={true}
+        initialFly={true}
+      />
+    </div>
+  );
+};
+
+export const MunicipalityIntro = ({ pageContext, className }) => {
+  const { slug } = pageContext;
+  let { municipality } = pageContext;
+  let type = municipality?.type;
+  const states = [
```

Comment:
Das hier vielleicht außerhalb der Komponente?

Reply:
Bin mir nicht sicher, ob ich das richtig verstanden habe (in der Email sah die zitierte Stelle länger aus und ich dachte du meintest die states). Meinst du slug, municipality und type schon in der Sections-Komponente bestimmen und dann weitergeben? – Dachte ich halte es eher kurz in der Sections-Komponente, da dort ja schon viel passiert. Aber wäre für mich auch ok :)